releasenotes/notes/transpile-617bd3a4e6f1c0d8.yaml

```
@@ -0,0 +1,10 @@
+---
+features:
+  - |
+    Adds `BaseExperiment._transpile` which is called to transpile the
```

This comment needs updating to refer to `_transpiled_circuits` method and slightly changed functionality (it returns a list of the transpiled circuits to be run on a backend).
```suggestion
    Adds :meth:`.BaseExperiment._transpiled_circuits` which is called to transpile the
```

---

Done.